tests/integration/node/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "0.0.12",
   "scripts": {
     "pretest": "node scripts/test.js",
-    "test": "mocha --opts mocha.opts \"test/**/*.spec.js\""
+    "test": "mocha --opts mocha.opts \"test/common/autostore.spec.js\""
```

Comment:
Is this temporary?

Reply:
This was fixed with https://github.com/Kinvey/js-sdk/pull/409/commits/04da399a69301965b65e950a33ae05133c91b959